client/components/QuestionsAndAnswers/AnswerItem.jsx

```diff
@@ -38,6 +38,10 @@ const AnswerItem = ({ answer, index, updateAnswerHelpfulness }) => {
         </h6>
```

Comment:
I think before I mentioned using span here, but I have a different reason why and you can disagree with me. I think from an accessibility standpoint it makes sense to use a span rather than multiple headings. I think if there was just one heading it would be fine, but it seems to be used more stylistically here rather than semantically.

Reply:
Totally forgot, I just made the changes, the code looks much better, thanks.